packages/testing/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@firebase/testing",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
We don't need to do this, even though it is a breaking change major `0`s allow for breaking changes between other versions also.

Reply:
done